docs/topics/installing-intellij-idea-plugin.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+[id="installing-intellij-idea-plugin_{context}"]
+= Installing the {ProductShortName} plugin for IntelliJ IDEA
+
+You can install the {ProductShortName} plugin in both IntelliJ IDEA Ultimate and the IDEA Community Edition.
+
+[discrete]
+include::jdk-hardware-mac-prerequisites.adoc[leveloffset=+2]
+
+.Procedure
+
+. Set the environmental variable `JAVA_HOME`:
++
+[source, terminal]
+----
+$ export JAVA_HOME=jdk11
+----
+
+. In IntelliJ IDEA, click *Plugins* icon on the Activity bar to open the *Plugins* view.
```

Comment:
*Plugins* icon > the *Plugins* icon

Reply:
Thanks -- I didn't realize the installation section was part of this PR. It shouldn't have been. that section is not ready for review!

Reply:
If installation is not part of this PR, you should remove it from the PR.